templates/patterns/file_teaser/pattern-file-teaser.html.twig

```diff
@@ -48,7 +48,7 @@
   },
   'translation': {
     'toggle': {
-      'label': "Available translations (@count)"|t({ '@count': translations|length}),
+      'label': translation_toggle_label ~ " (" ~ translations|length ~ ")",
```

Comment:
In order to make this backward compatible, i.e. when people are using patterns that don't have the new field set, let's do this:
```suggestion
      'label': translation_toggle_label|default("Available translations"|t) ~ " (" ~ translations|length ~ ")",
```